### ISSUE TITLE
Get the loss per epoch instead of only 500 and 1000 epochs

### ISSUE THREAD

**Issue by JVincentG** [EricFillion/happy-transformer#319]
Is there any way i can get the loss per step of epoch so i can generate a line chart of loss per epoch? thanks

**Comment by turinaf** [comment]
I am looking for this too. Please let us know if there's a way can get training loss per epochs so that we can visualize. 
Same for evaluaiton loss. @EricFillion 

Thank you in advance.

**Comment by EricFillion** [comment]
Thanks for the feedback.  This feature shouldn't be too hard to add. We just need to allow the user to adjust the  Hugging Face TrainingArguments's eval_steps parameter and possibly also the evaluation_strategy parameter as well. 

I'll look into it for the next update.

**Comment by JVincentG** [comment]
I just wanted to follow up on this feature. If you could guide me, @EricFillion, on where to look to update the HappyTransformer model, I would be willing to volunteer and create a pull request to close this issue. Thank you.

**Comment by EricFillion** [comment]
This feature was added in version 3.0.0. 

You can adjust how many times training occurs using the eval_steps argument for your TrainArgs class such as GENTrainArgs's _eval_steps_ parameter. Provide a float between 0 and 1 to specify the ratio of steps until evaluating occurs. By default it is 0.1 so evaluating occurs 10 times. 

```
from happytransformer import GENTrainArgs

args = GENTrainArgs(eval_steps=0.05)
```
